documentation/modules/exploit/linux/http/grandstream_ucm62xx_sendemail_rce.md

```diff
@@ -0,0 +1,220 @@
+## Vulnerable Application
+
+### Description
+
+This module exploits unauthenticated SQL and command injection vulnerabilities affecting
+the Grandstream UCM62xx IP PBX series of devices. The vulnerabilities allow an
+unauthenticated and remote attacker to execute commands as root. This module affects
```

Comment:
Do we not have CVE ID's for these vulnerabilities? Also is there any way to explain how these vulnerabilities tie together a bit more? It seems like this description as it stands glosses over a lot of important details in an effort to try make things short, but the result is that I'm really not sure what this module is meant to be exploiting and in what order to get its results from this description.

Reply:
Let me know if the update doesn't satisfy.